db/migrate/20140109053431_job_text_upgrade.rb

```diff
@@ -1,5 +1,5 @@
 class JobTextUpgrade < ActiveRecord::Migration
   def change
-    change_column :jobs, :output, :text, :limit => 1.gigabyte - 1
+    change_column :jobs, :output, :text, :limit => 1.gigabyte / 4 - 1
```

Comment:
nah ... even if there is they and us don't want 1GB of foobar just sitting
there

On Tue, Jun 2, 2015 at 9:25 PM, Roman Sandler <notifications@github.com>
wrote:

> In db/migrate/20140109053431_job_text_upgrade.rb
> <https://github.com/zendesk/samson/pull/434#discussion_r31592942>:
>
> > @@ -1,5 +1,5 @@
> >  class JobTextUpgrade < ActiveRecord::Migration
> >    def change
> > -    change_column :jobs, :output, :text, :limit => 1.gigabyte - 1
> > +    change_column :jobs, :output, :text, :limit => 1.gigabyte / 4 - 1
>
> 1Gb - 1byte is the max that postgresql allows.I'm a bit wary of reducing
> the limit in case there is old data (either ours or someone else's) that is
> too large
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/434/files#r31592942>.
>